website/_kb/user-guide/querying/favorites-tagging.md

```diff
@@ -0,0 +1,22 @@
+---
+category: querying
```

Comment:
This is relevant for dashboards too. Do you want to adapt the content to work for both? 

We will also need to figure how to place the same page in two categories (currently not supported).

Reply:
I did this the simplest way possible: see PR #161